examples_tests/39.DenoiserTonemapper/main.cpp

```diff
@@ -45,10 +45,12 @@ bool check_error(bool cond, const char* message)
 		os::Printer::log(message, ELL_ERROR);
 	return cond;
 }
-/*
-	if (check_error(,"!"))
-		return error_code;
-*/
+
+constexpr uint32_t overlap = 64;
+constexpr uint32_t tileWidth = 1920, tileHeight = 1080; //or 1920x1080
```

Comment:
let me know how much memory the denoiser says it needs for the scratch size with a 1080p tile

Reply:
470 081 920 bytes which is 448 MiB

Reply:
Try using 1024x1024 a 2048x2048 tile, it might expose logic errors when your test image is not divisible by tile size exactly